ArduPlane/quadplane.cpp

```diff
@@ -2950,6 +2950,8 @@ void QuadPlane::takeoff_controller(void)
     plane.nav_roll_cd = 0;
     plane.nav_pitch_cd = 0;
 
+    landing_detector->enabled();
```

Comment:
```suggestion
    plane.landing_detector.enabled();
```
You don't fill in the local pointer, so you have to go via plane.

Reply:
Thanks. It was `plane.g2.landing_detector.enabled()`;. Slowly learning the C++ architecture.